src/instructlab/lab.py

```diff
@@ -43,1332 +31,67 @@
     import torch
 
 
-class Lab:
-    """Lab object holds high-level information about ilab CLI"""
-
-    def __init__(self, config_obj: config.Config):
-        self.config = config_obj
-
-        # Set up logging for the Lab class
-        self.logger = logging.getLogger(__name__)
-
-        # Create a formatter
-        formatter = log.CustomFormatter(log.FORMAT)
-
-        # Create a handler and set the formatter
-        handler = logging.StreamHandler()
-        handler.setFormatter(formatter)
-
-        logging.basicConfig(
-            format=log.FORMAT,
-            handlers=[handler],
-        )
-
-        self.logger.setLevel(self.config.general.log_level.upper())
-
-
-@click.group(cls=DYMGroup)
-@click.option(
-    "--config",
-    "config_file",
-    type=click.Path(),
-    default=config.DEFAULT_CONFIG,
-    show_default=True,
-    help="Path to a configuration file.",
-)
+class ExpandAliasesGroup(click.Group):
+    def __init__(self, *args, **kwargs):
+        self.aliases = kwargs.pop("aliases", {})
+        super().__init__(*args, **kwargs)
+
+    def get_command(self, ctx, cmd_name):
+        if cmd_name in self.aliases:
+            cmd = self.aliases[cmd_name]["cmd"]
+            if cmd_name != "init":
+                config.init_config(ctx, config.DEFAULT_CONFIG)
+            return cmd
+        cmd = click.Group.get_command(self, ctx, cmd_name)
+        return cmd
+
+    def format_epilog(self, ctx, formatter):
+        """Inject our aliases into the help string"""
+        if self.aliases:
+            formatter.write_paragraph()
+            formatter.write_text("Aliases:")
+            with formatter.indentation():
```

Comment:
indentaion isn't working.   Would be nice to have indent after Aliases:

```
Aliases:
chat: model chat
convert: model convert
diff: taxonomy diff
download: model download
evaluate: model evaluate
generate: data generate
init: config init
list: model list
serve: model serve
train: model train

```

Reply:
https://github.com/instructlab/instructlab/issues/1324